AC3.2Unit2FinalAssessment/NumbersViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ViewController.swift
+//  AC3.2Unit2FinalAssessment
+//
+//  Created by Simone Grant on 10/6/16.
+//  Copyright © 2016 Simone Grant. All rights reserved.
+//
+
+import UIKit
+import Foundation
+
+class NumbersViewController: UIViewController {
```

Comment:
Needs to conform to UITextFieldDelegate and set `numberField.delegate = self` to be able to access textField methods.